src/Core/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ find_package(OpenMP REQUIRED)
 add_definitions(${OpenMP_CXX_FLAGS})
```

Comment:
by the way, this is very outdated and known to cause linking failures on some compilers. Best make `cilacc` depend on the `OpenMP::OpenMP_CXX` target.

Reply:
moved to #2155; perhaps related to #2154